src/walk.cpp

```diff
@@ -1,5 +1,67 @@
-// Generated by dust (version 0.7.12) - do not edit
-#include <iostream>
+// Generated by dust (version 0.8.0) - do not edit
+#include <cpp11.hpp>
+[[cpp11::register]]
+SEXP dust_walk_alloc(cpp11::list r_pars, bool pars_multi, size_t step,
+                         cpp11::sexp r_n_particles, size_t n_threads,
+                         cpp11::sexp r_seed, cpp11::sexp device_id);
```

Comment:
Do any of the examples define a run_device()?

Reply:
inst/examples/variable.cpp does